### ISSUE TITLE
Big Sur guest idle uses a lot of CPU on host

### ISSUE THREAD

**Issue by igorzharii** [foxlet/macOS-Simple-KVM#413]
Hey, did anyone notice Big Sur eating a lot of CPU of the host even on idle? Catalina didn't do that, does anyone else have this issue?

**Comment by funderscore1** [comment]
I don't think big sur is supported by this YET

**Comment by ghost** [comment]
were you able to get bigsur working ?

**Comment by igorzharii** [comment]
Yes, it's working but on idle is using more of host's CPU than Catalina..

**Comment by mwlczk** [comment]
> Yes, it's working but on idle is using more of host's CPU than Catalina..

what Product ID did you use for Big Sur, i guess they changed a couple of times as the developer seed evolved? 
And is the UI laggy or is it still smooth like under catalina?

**Comment by igorzharii** [comment]
Not sure about the product ID - because I actually used [kholia's](https://github.com/kholia/OSX-KVM) version which has Big Sur but no issues/comments, so I decided to comment here, assuming both versions are similar. 

In that solution the UI speed is similar to Catalina's, but I'm using it as VM and only via VNC so unfortunately cannot tell you how smooth the UI is while connected to a monitor.
